### PR TITLE
Update GuzzleHttp/guzzle dependency from ^6.5 to ^7.0

### PR DESCRIPTION
**Pull request by Abouhassane**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php": ">=7.2",
-        "guzzlehttp/guzzle": "^6.5",
+        "guzzlehttp/guzzle": "^7.0",
```
Comment (atymic) [inline composer.json:16]: Is it possible to support both versions? Guessing a lot of people will still be on 6.x

Comment (Abouhassane) [reply composer.json:16]: I think we can do that, the same you did to `illuminate/notification` and `illuminate\support`:

> "illuminate/notifications": "~5.5 || ~6.0 || ~7.0"
> "illuminate/support": "~5.5 || ~6.0 || ~7.0"

Would you like me to make another pull request of that?

Comment (atymic) [reply composer.json:16]: Sure, just update the PR.

Comment (atymic) [reply composer.json:16]: Check: https://github.com/guzzle/guzzle/blob/master/UPGRADING.md

**Pull request merged**